ansible/mint_data.yml

```diff
@@ -8,24 +8,8 @@
       when: registers is none
       always_run: yes
 
-- name: Compute mint hosts
-  hosts: localhost
-  vars:
-    vpc_tag: 'tag_Environment_{{ vpc }}'
-  tasks:
-    - name: query openregister hosts
-      set_fact:
-        openregister_hosts: "{{ groups.tag_Role_openregister_app | intersect(groups[vpc_tag]) }}"
-      always_run: yes
-
-    - name: include matching hosts
-      add_host: name="{{ item }}" group="collected_hosts"
-      with_items: "{{ openregister_hosts }}"
-      when: hostvars[item].ec2_tag_Register == registers
-      always_run: yes
-
 - name: Load data
-  hosts: "collected_hosts[0]"
+  hosts: "{{ registers }}-1.{{ vpc }}.openregister"
```

Comment:
What happens if the `registers` variable is a list? I believe this may have been broken in the previous commit as well.

The deployment README suggests that a list of register names can be passed. We should either fix this or change the variable name to `register` and update the README.